.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "singleQuote": true,
+  "printWidth": 120,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "bracketSpacing": true,
+  "semi": false,
+  "useTabs": false,
+  "jsxBracketSameLine": false
+}
```

Comment:
Please keep the same file extension `.prettierrc.json` for prettier config

Reply:
So, as I understand,  .prettier.json now can be changed to 
{
  "htmlWhitespaceSensitivity": "ignore",
  "semi": false,
  "useTabs": false,
  "jsxBracketSameLine": false,
  "overrides": [
    {
      "files": ["*.css", "*.scss"],
      "options": {
        "printWidth": 100
      }
    }
  ]
}

Reply:
Let's keep the defaults for now. Files still are autoformatted.

Reply:
No need to make any changes to prettier config files